PyInstaller/hooks/hook-gi.repository.Gio.py

```diff
@@ -0,0 +1,37 @@
+import glob
+import os
+import sys
+
+import PyInstaller.log as logging
+from PyInstaller.compat import is_darwin, is_win
+from PyInstaller.utils.hooks import get_typelibs, exec_statement
+
+logger = logging.getLogger(__name__)
+
+
+hiddenimports = ['gi.overrides.Gio']
+
+
+datas = get_typelibs('Gio', '2.0')
+
+
+binaries = []
+
+statement = """
+from gi.repository import Gio
+print(Gio.__path__)
+"""
+
+path = exec_statement(statement)
+pattern = None
+
+if is_darwin:
+    pattern = os.path.join(os.path.commonprefix([sys.prefix, path]), 'lib', 'gio', 'modules', '*.so')
```

Comment:
On Mac OS the path to the Gio module is `/opt/local/lib/girepository-1.0/Gio-2.0.typelib`, `sys.prefix` is `/opt/local/Library/Frameworks/Python.framework/Versions/3.4`, and the path to the Gio modules is `/opt/local/lib/gio/modules/` so the `commonprefix` would be `/opt/local/`. This should also hold try if everything were installed in `/usr/local/`.

On Windows the path to the Gio module is `C:\Python34\Lib\site-packages\gnome\lib\girepository-1.0\Gio-2.0.typelib`, `sys.prefix` is `C:\Python34`, and the path to the Gio modules is `C:\Python34\Lib\site-packages\gnome\lib\gio\modules\` so in this case the `commonprefix` would be `C:\Python34` which wouldn't get us the right `lib` directory.